go/internal/sidecar/responses_pipeline.go

```
@@ -0,0 +1,212 @@
+	"strings"
+
+	"github.com/lidge-jun/opencodex/go/internal/jsonwire"
+)
+
+type responseRepairPipeline struct {
+	modelID      string
+	imageAliases map[string]imageAlias
+	reasoning    bool
+	itemIDs      *itemIDRepairState
+	snapshot     *snapshotRepairState
+	request      *jsonwire.Value
```

golangci-lint (2.13.2)

[error] field request is unused

(unused)